### ISSUE TITLE
3 Slides Centered with the Previous and Next Slides Partially Opaque

### ISSUE THREAD

**Issue by svoltmer** [stevenwanderski/bxslider-4#617]
I can't seem to find a way to have one slide totally visible and centered with both the previous and next slides off the sides of the centered current slide, but partially opaque. Any help would be appreciated.

**Comment by cleanboy** [comment]
Are you referring to something like this?

http://www.aucklanddj.co.nz/weddings

I believe they are using bxslider but the code on that site is minified and difficult to read.

Where the previous and next slides go off the page? I am trying to achieve something similar but when I set slideWidth to anything bigger than the maximum screen size it sets a maximum size so they all fit in. I cant seem to figure out a way around this...

**Comment by franbueno** [comment]
I would say, not at all, because its not responsive :-1:

**Comment by cleanboy** [comment]
Yeah...this is true...for the project im working on I ended up having to create a second mobile slideshow which is hidden on the desktop version...not the most efficient way but the best I could think of...im open to suggestions though!
